src/style/themes/base/base-theme.config.js

```diff
@@ -21,9 +21,9 @@ export default (palette) => {
       // status
       error: palette.errorRed,
       focus: palette.gold,
-      info: palette.productBlueShade(3),
-      success: palette.brilliantGreenShade(20),
-      warning: palette.gold
+      info: palette.infoBlue,
```

Comment:
These values, and the values in `style/colour-config/` should not be changed or renamed.